source/Agents/GP/GPAgentTest.hpp

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file agent.hpp
```

Comment:
I haven't been paying enough attention to these, but we should make sure these headers are accurate